src/main/java/duke/request/Command.java

```diff
@@ -0,0 +1,47 @@
+package duke.request;
+
+import duke.exception.UserException;
+
+/**
+ * A duke.request.Command represents a string in a duke.request.Request that instructs the application to
+ * perform an Action.
+ */
+public enum Command {
+    BYE("bye"),
+    FIND("find"),
+    LIST("list"),
+    DONE("done"),
+    DELETE("delete"),
+    DEADLINE("deadline"),
+    EVENT("event"),
+    TODO("todo");
+

```

Comment:
Great that you used enum. Makes the entire code a lot more concise and easier to read. 👍

Reply:
Thanks!